src/components/GlobalPreferences/GlobalPreferences.js

```diff
@@ -243,13 +242,15 @@ function AnimatedGlobalPreferences(props) {
               ),
             }}
           >
-            <GlobalPreferences
-              {...props}
-              compact={compact}
-              sectionIndex={sectionIndex}
-              subsection={subsection}
-              onNavigation={handleNavigation}
-            />
+            <Layout paddingBottom={0}>
+              <GlobalPreferences
+                {...props}
+                compact={compact}
+                sectionIndex={sectionIndex}
+                subsection={subsection}
+                onNavigation={handleNavigation}
+              />
+            </Layout>
```

Comment:
`GlobalPreferences` already declares `Layout` internally and I think we can remove this.

Reply:
Yes!